docs/MigratingToV3.md

```diff
@@ -0,0 +1,242 @@
+# Migrating to version 3
+
+In version 3 the `natds-styles` theme library was replaced with `natds-themes`.
+This change is important because now this library depends on a theme build specifically for react-native that share values with android and ios. Previously, this was done by transforming the theme that was build for web and material ui.
+
+For more information about natds-themes can be found [here](https://github.com/natura-cosmeticos/natds-commons).
```

Comment:
Aqui não precisa do 'For', né? Acho que você começou a frase pensando de um jeito, mudou e esqueceu ele aí :D
E aproveita bota os \` \` em volta do nome `natds-themes`

Reply:
HAHAHAHAH foi totaaaal isso que aconteceu :P